static/app/views/ddm/contextMenu.tsx

```diff
@@ -0,0 +1,107 @@
+import {InjectedRouter} from 'react-router';
+import styled from '@emotion/styled';
+
+import {openAddToDashboardModal} from 'sentry/actionCreators/modal';
+import {DropdownMenu} from 'sentry/components/dropdownMenu';
+import {IconEllipsis} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Organization} from 'sentry/types';
+import {MetricDisplayType, MetricsQuery} from 'sentry/utils/metrics';
+import useOrganization from 'sentry/utils/useOrganization';
+import useRouter from 'sentry/utils/useRouter';
+import {DashboardWidgetSource} from 'sentry/views/dashboards/types';
+
+type ContextMenuProps = {
+  metricsQuery: MetricsQuery;
+  displayType?: MetricDisplayType;
+};
+
+export function MetricWidgetContextMenu({metricsQuery, displayType}: ContextMenuProps) {
+  const organization = useOrganization();
+  const router = useRouter();
+
+  if (!organization.features.includes('ddm-experimental')) {
+    return null;
+  }
+
+  return (
+    <StyledDropdownMenuControl
+      items={[
+        {
+          key: 'add-alert',
+          label: t('Create Alert'),
+          disabled: true,
+        },
+        {
+          key: 'add-dashoard',
+          label: t('Add to Dashboard'),
+          onAction: () => {
+            handleAddQueryToDashboard(metricsQuery, organization, router, displayType);
+          },
+        },
+      ]}
+      triggerProps={{
+        'aria-label': t('Widget actions'),
+        size: 'xs',
+        borderless: true,
+        showChevron: false,
+        icon: <IconEllipsis direction="down" size="sm" />,
+      }}
+      position="bottom-end"
+    />
+  );
+}
+
+function handleAddQueryToDashboard(
+  {projects, environments, datetime, op, mri}: MetricsQuery,
+  organization: Organization,
+  router: InjectedRouter,
+  displayType?: MetricDisplayType
+) {
+  const {start, end, period} = datetime;
+  // TODO(ddm): make a util that does this
+  const field = op ? `${op}(${mri})` : mri;
```

Comment:
Out of curiosity: do we support querying MRIs without operator?
I already have a similar util for alerts, can move it to ddm.

Reply:
I don't think we do, maybe we should throw an error or use a default if that can even happen. And yep, move the util to ddm then and i will reuse it here

Reply:
ATM I handle it as if no MRI was selected -> the create action is disabled.

Reply:
makes sense, no op no party :D